### ISSUE TITLE
[Suggestion] Shaft Power Transfer with Mekanism Teleporter Portals / RFTools Matter Transmitter/Receiver

### ISSUE THREAD

**Issue by amesgen** [ReikaKalseki/Reika_Mods_Issues#1038]
As the DE Portals can't be supported, what about the [Mekanism ones](http://wiki.aidancbrady.com/wiki/Teleporter_Portal)? Once again, a few reasons:
- new mod integration
- more expensive than Mystcraft portals (+ recipe tweaking leads to the ability of gating interdimensianl shaft power transport behind certain materials)
- the Mekanism portals can use private channels -> nice for servers
- as the Mekanism portals use RF, you could add a tax for using it for shaft power transport (some RF/t or increased power usage when teleporting as a player)
- Mekanism exists for newer minecraft versions, so this integration would be a "long-term"-one
- the channels of these portals can be accessed and edited by computers (CC/OC), which might result in interesting possibilities

[As  Dimmerworld said](https://github.com/ReikaKalseki/Reika_Mods_Issues/issues/1038#issuecomment-235774638), the same thing **could** be done with the RFTools teleportation system.

**Comment by Saereth** [comment]
This would be a really nice addition if its possible

**Comment by jusvit** [comment]
Rftools would also be nice with teleporters or Worlds

**Comment by amesgen** [comment]
Yes, and the teleporters occur as a "pair" (one Matter Transmitter points to one Matter Receiver).
Maybe @McJty and @aidancbrady can say sth about these suggestions?

**Comment by McJty** [comment]
Well I'm not adding new features to 1.7.10 anymore as I have fully moved to MC 1.10.2 and Reika (as far as I know) hasn't started moving to that yet so that basically makes this a no-go at this point

**Comment by TomeWyrm** [comment]
Closing this one for now. It's not really an issue that can be worked on in the present or near future. Someone will either bring the issue back up when Reika updates, or one of us will notice/remember and re-open the issue.

**Comment by ReikaKalseki** [comment]
I say leave it open, as its fundamental point has not been addressed. That said, a new label is appropriate.
